packages/clay-card/src/__tests__/index.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import * as TestRenderer from 'react-test-renderer';
+
+import ClayCard from '../index';
+
+import CardFile from '../../fixtures/file/CardFile';
```

Comment:
I think using these from `fixtures` is overly complex and it makes it more difficult when viewing the source via storybook. This may be alleviated if we decide to go with high-level components(see discussion [here](https://github.com/liferay/clay/pull/2017)). If we go that route, most of these could be high-level components.

Reply:
Solved